lib/moonshot/stack.rb

```diff
@@ -195,6 +198,24 @@ def load_parameters_file
       end
     end
 
+    # @return [String] the path to the parameters file without extension.
+    def raw_template_file_name
+      @raw_template_file_name ||= File.join(Dir.pwd, 'cloud_formation', @app_name)
+    end
+
+    def load_template_file
+      json_template = JsonStackTemplate.new("#{raw_template_file_name}.json", log: @log)
+      yaml_template = YamlStackTemplate.new("#{raw_template_file_name}.y*ml", log: @log)
```

Comment:
We already dictate that parameters files end in .yml, without support for .yaml -- I'd prefer to just do the same here. It simplifies documentation and code.